pkg/storage/minio/minio.go

```diff
@@ -110,13 +110,23 @@ func (c *Client) SaveFile(bucket, filePath string) error {
 	if err != nil {
 		return err
 	}
+
 	var fileName string
 	if strings.Contains(filePath, "/") {
-		fileName = filePath
+		// fileName = filePath
+
+		// strip the filename if the path is longer than 3 subdirectories /dir/sub1/sub2/ -> /sub1/sub2
+		split := strings.Split(filePath, "/")
```

Comment:
isn't it better to use path.Split()? https://pkg.go.dev/path#Split

Reply:
yes it would